tests/net/lib/dns_addremove/prj.conf

```diff
@@ -0,0 +1,28 @@
+
+CONFIG_NETWORKING=y
+CONFIG_NET_TEST=y
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+CONFIG_NET_L2_DUMMY=y
+
+CONFIG_DNS_RESOLVER=y
+CONFIG_DNS_RESOLVER_MAX_SERVERS=2
+CONFIG_DNS_NUM_CONCUR_QUERIES=1
+
+CONFIG_NET_LOG=y
+
+CONFIG_NET_MGMT=y
+CONFIG_NET_MGMT_EVENT=y
+CONFIG_NET_MGMT_EVENT_STACK_SIZE=768
+CONFIG_NET_MGMT_EVENT_THREAD_PRIO=7
+CONFIG_NET_MGMT_EVENT_QUEUE_SIZE=2
+
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=n # IPv6 support is not currently available

```

Comment:
I did not quite understand you comment "IPv6 event names do not match as I ported the code over from code I was developing but there are currently no IPv6 events."
Do you know what is the actual issue? The DNS does not really depend on IPv4/IPv6 so it should work on top of both protocols. So it would be great if we could get rid of separate ifdef's in the tests.

Reply:
I haven't tested with IPv6, I'll give it a try